input/tangy-timed.js

```diff
@@ -480,20 +485,40 @@ class TangyTimed extends PolymerElement {
 
     }
   }
-
+  shouldGridAutoStop() {
+    console.log(this.autoStop)
+    const isSetsEqual = (a, b) => a.size === b.size && [...a].every(value => b.has(value));
+    const tangyToggleButtons = [].slice.call(this.shadowRoot.querySelectorAll('tangy-toggle-button'))
+    if (!tangyToggleButtons[0].__data.pressed) {
```

Comment:
Was `tangyToggleButtons[0].pressed` not working here?

Reply:
Didnt see it in the list of properties. I will try that